sample/config.php

```diff
@@ -8,6 +8,5 @@
 // Since the API accepts only integer values, the currencyBaseUnitMultiplier is used convert the decimal amount into the accepted base units integer value.
```

Comment:
Why would these files get changed - thought we only need to add the property file

Reply:
the branch/PR was originally based off of master. We aren't using master in web and actually referencing `always_verify_ssl_cert_and_host` which is 29 commits behind master. I went ahead and rebased the branch off so the changes should just reflect the appsec.properties file now